new-dti-website/components/footer.tsx

```diff
@@ -43,7 +44,7 @@ const socialIcons: Icon[] = [
 const Footer: React.FC = () => (
   <div className="w-full h-[146px] inset-x-0 bottom-0 md:px-[40px] lg:px-[60px] inline-flex justify-between items-center md:flex-row flex-col">
     <div className="sm: w-[336px] sm:text-center md:text-lg md:w-[310px] md:text-left lg:w-full text-neutral-50 text-sm font-medium flex items-center ">
-      © 2023 Cornell Digital Tech & Innovation Project Team
+      © {config.semester.split(' ')[1]} Cornell Digital Tech & Innovation Project Team
```

Comment:
Maybe `new Date().getFullYear()`?  Since that's more up-to-date and allows us to avoid this string parsing here.